src/main/java/com/mattermost/integration/figma/config/exception/handler/ExceptionHandlingController.java

```diff
@@ -56,4 +56,10 @@ public String handleMMSubscriptionFromDMChannelException(
             RuntimeException ex, WebRequest request) {
         return String.format("{\"type\":\"error\",\"text\":\"%s\"}", ex.getMessage());
     }
+
+    @ExceptionHandler(value = {FigmaCannotCreateWebhookException.class})
+    @ResponseBody
```

Comment:
maybe just add FigmaCannotCreateWebhookException.class to  handleMMSubscriptionFromDMChannelException(
             RuntimeException ex, WebRequest request) 
and pls rename it to handleErrors